glide.yaml

```diff
@@ -32,8 +32,10 @@ import:
   - lib/options
   - lib/upgrade/migrator
   - lib/upgrade/migrator/clients
+# gobuffalo/logger needs the SetOutput function in the Logger type, so pin
+# the first version that introduces it.
 - package: github.com/sirupsen/logrus
-  version: v1.0.4
+  version: v1.0.6
```

Comment:
@fasaxc didn't we pin to v1.0.4 specifically since newer versions break some of our logging code?

Do we need to do more here before we can rev this?

Reply:
From OOB conversation with @fasaxc, the issue (https://github.com/sirupsen/logrus/issues/729) seems to be fixed in 1.0.6 (https://github.com/sirupsen/logrus/commit/070c81def33f6362a8267b6a4e56fb7bf23fc6b5) so we should be fine here.

Reply:
Hm, I'm not seeing the fix work, though I might be doing something wrong. Here's some output from a Felix FV log using the new logrus:

```
[INFO][11] <nil> <nil>: Parsing value for LogSeverityFile: None (from config file)
```

Reply:
Sigh

Reply:
All the logging using formatting versions of the functions (`Debugf`, `Infof`, `Warnf`) ended up having `<nil> <nil>` for some reason. Using `Debug` or `Info` works just fine. So I suppose the issue is somewhere else in logrus.

Fix one thing, break another. :)